docs/src/main/asciidoc/security-customization.adoc

```diff
@@ -107,13 +109,13 @@ public class CustomAwareJWTAuthMechanism implements HttpAuthenticationMechanism
 
 	@Override
 	public Set<Class<? extends AuthenticationRequest>> getCredentialTypes() {
-		return selectBetweenJwtAndOidc(context).getCredentialTypes();
+		return selectBetweenJwtAndOidc(null).getCredentialTypes();
```

Comment:
I just prototyped some code - but this specific update shows it was a bad prototype :-). I think here we need to return a combination of 2 sets returned from JWT and OIDC...

Reply:
Done. BTW I built docs to be sure - the example is aligned.